src/AzureOperator.ts

```diff
@@ -6,46 +6,18 @@
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
 import {TelemetryContext} from './telemetry';
-import {ProjectHostType} from './Models/Interfaces/ProjectHostType';
-import {handleIoTWorkspaceProjectFolder, askAndNewProject} from './utils';
+import {createAndLoadIoTProject} from './utils';

```

Comment:
Why not import all?

Reply:
WIll change to `import {handleIoTWorkspaceProjectFolder, askAndNewProject, createAndLoadIoTProject} from './utils';`